### PR TITLE
Add alternative font stack for extended-Latin languages [fix #15687]

### LINKED ISSUE #15687: Refresh: update localized font stacks
## Description

The new brand fonts don't currently include a number of Latin Extended characters. Some languages that rely on the extended alphabet such as Turkish, Czech, and Vietnamese (among others) will show most characters in the brand font, but unsupported characters will render in the next available fallback font. It's readable, but unattractive and annoying to see one letter in the middle of a word rendered in a very different font.

We should use our l10n CSS capabilities to deliver the brand fonts only to those languages we know it supports, and other languages can use a default font stack. We can use Inter as the primary webfont for those.

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Our new brand fonts are missing a few extended characters necessary to support a few languages that use Latin alphabets, resulting in annoying mixed fonts. It's readable but doesn't look great.

image

Hopefully we can get those characters added in the near future but until then it's preferable to override the brand font for these languages: Croation, Czech, Hungarian, Latvian, Lithuanian, Romanian, Slovak, Slovenian, Turkish, Vietnamese, and Welsh. For those languages we can fall back to Inter as a preferred webfont (it has a really broad character set).

I'm using the version of Inter that comes from Protocol, but needed to add a semibold weight to bedrock since Protocol doesn't currently provide that. We could add all the weights/styles to bedrock but that seemed redundant. 

## Issue / Bugzilla link
#15687 

## Testing
The manifesto page is a good one to test with since it's widely localized and is a lot of text.

http://localhost:8000/hr/about/manifesto/
http://localhost:8000/cs/about/manifesto/
http://localhost:8000/hu/about/manifesto/
http://localhost:8000/lv/about/manifesto/
http://localhost:8000/lt/about/manifesto/
http://localhost:8000/ro/about/manifesto/
http://localhost:8000/sk/about/manifesto/
http://localhost:8000/sl/about/manifesto/
http://localhost:8000/tr/about/manifesto/
http://localhost:8000/vi/about/manifesto/
http://localhost:8000/cy/about/manifesto/

### CODE REVIEW

**Comment by janbrasna** [comment]
Good catch with Welsh, I totally missed that one, even though checking for it… 🙈  Is this list supposed to be exhaustive, or it's just the most visible/disruptive ones?

There's also few [Turkish](https://www.mozilla.org/tr/about/manifesto/) accents missing as well as some Berber [Taqbaylit](https://www.mozilla.org/kab/about/manifesto/), and then a few more Slavic from prod, as [Bosanski](https://www.mozilla.org/bs/about/manifesto/), [Dolnoserbšćina](https://www.mozilla.org/dsb/about/manifesto/) and [Hornjoserbsce](https://www.mozilla.org/hsb/about/manifesto/)…

**Comment by craigcook** [comment]
> Good catch with Welsh, I totally missed that one, even though checking for it… 🙈 Is this list supposed to be exhaustive, or it's just the most visible/disruptive ones?
> 
> There's also few [Turkish](https://www.mozilla.org/tr/about/manifesto/) accents missing as well as some Berber [Taqbaylit](https://www.mozilla.org/kab/about/manifesto/), and then a few more Slavic from prod, as [Bosanski](https://www.mozilla.org/bs/about/manifesto/), [Dolnoserbšćina](https://www.mozilla.org/dsb/about/manifesto/) and [Hornjoserbsce](https://www.mozilla.org/hsb/about/manifesto/)…

Not exhaustive, so I'm sure I missed some. I'll add them. And I can't believe I skipped Turkish, which was one of the original ones that came to our attention 🤦  There are also a few languages we don't currently support on mozorg so I left them out intentionally.

**Comment by craigcook** [comment]
Oh hang on, I didn't miss Turkish, I just omitted it from the list in the description and from the test image, but it's included in the PR.

**Comment by janbrasna** [comment]
I blindly trusted the description;)

One more I found is gn-Guarani (Avañe'ẽ) that misses its ẽ, ỹ, ũ

File: media/css/l10n/ro/intl.css [hunk 1]
```diff
@@ -1,91 +1,13 @@
-/* Bug 907793, 1263732 */
-
-@font-face {
```
[the remark this answers is not in the document]
Comment (craigcook) [reply media/css/l10n/ro/intl.css:3]: Yeah, those should be safe to delete.

**Comment by janbrasna** [comment]
(I have finally noticed I need to rebuild to see all the changes correctly, and now can confirm Turkish works for me too… The `az` also looks good, no signs of mplus missing, as well as any old custom `ro` fonts.)

I did test the KO word break, and also confirmed it has no adverse effects on other CJK locales.

**Review by maureenlholland — APPROVED** [review]
Inter across the board r+

**Pull request merged**